CHANGELOG.md

```diff
@@ -27,6 +27,7 @@
 * [CHANGE] Distributor: Replace `-distributor.retry-after-header.max-backoff-exponent` and `-distributor.retry-after-header.base-seconds` with `-distributor.retry-after-header.min-backoff` and `-distributor.retry-after-header.max-backoff` for easier configuration. #8694
 * [CHANGE] Ingester: increase the default inactivity timeout of active series (`-ingester.active-series-metrics-idle-timeout`) from `10m` to `20m`. #8975
 * [CHANGE] Distributor: Remove `-distributor.enable-otlp-metadata-storage` flag, which was deprecated in version 2.12. #9069
+* [CHANGE] Ruler: `-ruler.drain-notification-queue-on-shutdown` is now enabled by default, and no longer considered experimental.
```

Comment:
```suggestion
* [CHANGE] Ruler: `-ruler.drain-notification-queue-on-shutdown` is enabled by default and no longer experimental.
```

Reply:
Thanks. I've modified the changelog entry, since we have removed the option completely. PTAL.